### DISCUSSION TITLE
GitHub error message: Resource not accessible by integration

### DISCUSSION THREAD

**Discussion by wxyucs** [Mergifyio/mergify#2474] [Q&A] [answered]
Hi all,
I got the error "GitHub error message: Resource not accessible by integration in pull requests below:
https://github.com/milvus-io/milvus/pull/5046/checks?check_run_id=2435949005
https://github.com/milvus-io/pymilvus-orm/pull/56/checks?check_run_id=2430752155

It seems all conditions are satisfied, but Mergify is failed to merge the pull requests. (In other pull requests, Mergify works well) 
How to fix it? 
Thanks

**Comment by sileht** [comment, marked as answer]
Did you restrict who can push to your main branch? If yes, you may need some additional GitHub configuration:
https://docs.mergify.io/faq/#mergify-is-unable-to-merge-my-pull-request-due-to-my-branch-protection-settings

**Reply by wxyucs** [reply]
Thanks for your reply. I can be sure that "Restrict who can push to matching branches" is disabled. And it's only failed in these two pull requests, after that, works well.

**Reply by wxyucs** [reply]
The error appeared again: 
https://github.com/milvus-io/milvus/pull/5067/checks?check_run_id=2453675844

Where can I get some more detail?

**Reply by wxyucs** [reply]
It seems that after the base branch updated the contents of the .github/workflow directory, the head branch did not synchronize the base branch. After merging the base branch, this error dismissed.
